### PR TITLE
Updates to stretch cluster

### PR DESCRIPTION
**Pull request by dcuomo**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@dcuomo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by v-dasis** [comment]
looks good - thanks!

**Comment by v-dasis** [comment]
#sign-off

File: azure-stack/hci/concepts/host-network-requirements.md [hunk 1]
```diff
@@ -277,17 +277,24 @@ Stretched clusters provide disaster recovery that spans multiple data centers. I
 Stretched clusters have the following requirements and characteristics:
 
 - RDMA is limited to a single site, and is not supported across different sites or subnets.
+
 - Servers in the same site must reside in the same rack and Layer-2 boundary.
-- Communication between sites cross a Layer-3 boundary; stretched Layer-2 topologies are not supported.
 
-- If a site uses RDMA for its storage adapters, these adapters must be on a separate subnet and VLAN that cannot route between sites. This prevents Storage Replica from using RDMA across sites.
+- Communication between sites must cross a Layer-3 boundary; stretched Layer-2 topologies are not supported.
```
Comment (dcuomo) [inline azure-stack/hci/concepts/host-network-requirements.md:283]: While you're at it, this should probably say "Host communication between sites..."

File: azure-stack/hci/concepts/host-network-requirements.md [hunk 2]
```diff
@@ -277,17 +277,24 @@ Stretched clusters provide disaster recovery that spans multiple data centers. I
...
+
+- Has enough bandwidth to run the workloads at the other site. In the event of a failover, the alternate site will need to run all traffic. It is recommended to provision sites at 50% of their available network capacity. This is not a hard requirement if you are able to tolerate lower performance during a failover.
+
+- Replication (north/south) between sites can use the same physical NICs as the local storage (east/west which may use RDMA). If using the same physical adapters, these adapters must be teamed with SET, and must have additional virtual NICs provisioned between sites (RDMA may be enabled on the local virtual NICs; disabled on the routable virtual NICs).
```
Comment (dcuomo) [inline azure-stack/hci/concepts/host-network-requirements.md:287]: This is a bit confusing when thinking about the actual requirements. Please change this:
"local storage (east/west which may use RDMA)."

To this to simplify and clarify that N/S does not use RDMA: "local storage (east/west)."

This:
must have additional virtual NICs provisioned between sites (RDMA may be enabled on the local virtual NICs; disabled on the routable virtual NICs).

to this:
must have additional virtual NICs provisioned between sites (RDMA must be disabled on the routable virtual NICs).

**Pull request merged**